ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -182,13 +185,14 @@ class IdpLoginScreen extends React.Component<Props, State> {
                 : "authentication.errors.login.title"
             )}
           </Text>
-          <Text style={styles.errorBody}>
-            {I18n.t(
-              errorType === "LOADING_ERROR"
-                ? "authentication.errors.network.body"
-                : "authentication.errors.login.body"
+          {errorType === "LOGIN_ERROR" &&
+            this.state.errorCode && (
```

Comment:
you should show a message error also when it hasn't an error code
<img src="https://user-images.githubusercontent.com/822471/61730833-832eab80-ad7a-11e9-98b0-86a0f1a3e898.png" width="300">

Reply:
Fix in: 92f9d68